src/customErrors.ts

```diff
@@ -49,7 +49,7 @@ export class Exception_FindReplaceIllegalAction extends BaseException {
  * ```
  */
 interface ClassBuilder<T = BaseException> {
-  new (msg: string): T;
+  new(msg: string): T;
```

Comment:
Not sure why this is changed. Perhaps a forgotten lint rule in which case your editor is stepping in with its conventions.
In any case, I'll have to look into this.

edit: spelling